<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>WorldWindow</class>
 <widget class="QMainWindow" name="WorldWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>460</width>
    <height>368</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Ignored" vsizetype="Ignored">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>Mundo</string>
  </property>
  <property name="windowIcon">
   <iconset resource="resources.qrc">
    <normaloff>:/new/prefix1/worldwide.png</normaloff>:/new/prefix1/worldwide.png</iconset>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="sizePolicy">
    <sizepolicy hsizetype="Ignored" vsizetype="Ignored">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="whatsThis">
    <string>Vertical Layout
</string>
   </property>
   <widget class="QWidget" name="verticalWidget" native="true">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>211</width>
      <height>211</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>100</width>
      <height>100</height>
     </size>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QPushButton" name="generatePeople">
         <property name="text">
          <string>Generar</string>
         </property>
         <property name="autoDefault">
          <bool>false</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPlainTextEdit" name="people">
         <property name="enabled">
          <bool>true</bool>
         </property>
         <property name="sizePolicy">
          <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="minimumSize">
          <size>
           <width>10</width>
           <height>10</height>
          </size>
         </property>
         <property name="backgroundVisible">
          <bool>false</bool>
         </property>
         <property name="placeholderText">
          <string>Cantidad a crear..</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_3">
       <item>
        <widget class="QPushButton" name="generateSins">
         <property name="text">
          <string>Pecar</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="generateGoodDeeds">
         <property name="text">
          <string>Buenas Acciones</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QPushButton" name="top10sins">
       <property name="text">
        <string>Paises más pecadores</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="top5sins">
       <property name="text">
        <string>Paises menos pecadores</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="top10GD">
       <property name="text">
        <string>Paises con más buenas acciones</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="top5GD">
       <property name="text">
        <string>Paises con menos buenas acciones</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="humansCount">
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QListWidget" name="list">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>210</y>
      <width>431</width>
      <height>151</height>
     </rect>
    </property>
   </widget>
   <widget class="QWidget" name="verticalWidget_2" native="true">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>0</y>
      <width>211</width>
      <height>100</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>100</width>
      <height>100</height>
     </size>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_2">
     <item>
      <widget class="QLabel" name="label">
       <property name="text">
        <string>Consultas del infierno</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="showDemon">
       <property name="text">
        <string>Demonios</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton_2">
       <property name="text">
        <string>Humanos más pecadores</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
 </widget>
 <resources>
  <include location="resources.qrc"/>
 </resources>
 <connections/>
</ui>
